src/controllers/UserRequests.js

```diff
@@ -33,151 +85,54 @@ export function sendUserInfo() {
-//     //id: "15", //constant id
-//     //id: transcriptOutput, //variable id
-//     apiKey: submittedApiKey,
-//     apiVersion: submittedApiVersion,
-//     //???????
-//     //prompt: activeTab,
-//   });
-
-//   //create the fetch request
-//   fetch(`http://localhost:8080/api/v1/transcripts/storeAPI`, {method: "POST", headers: headerData, body: fullEntry})
-//   .then((response) => response.json())
-//   .then((data) => {
-//     console.log(data);
-//   });
-
-//   //empty the textboxes (maybe only do this if the submission fails do to an error or no transcripts or something else?)
-//   document.getElementById("apiKey").value = "";
-//   document.getElementById("versionID").value = "";
-// }
-
-// export function sendAccount() {
-//   //for debugging
-//   //alert('api sent');
-//   //console.log('api sent???');
-
-//   //retrieve the text content from the corresponding textbox.
-//   var submittedAddress = document.getElementById("emailAddress").value;
-//   var submittedPassword = document.getElementById("password").value;
-//   var submittedDiagramID = document.getElementById("diagramID").value;
-
-//   //alert(submittedApiKey + ', '+ submittedApiVersion);
-
-//   //the json data sent to the backend
-//   //JSON.stringify is required for fetch
-//   const fullEntry = JSON.stringify({
-//     //if i don't provide an id, mongo will create one called Object(...). IDs have to be unique!
-//     //id: "15", //constant id
-//     //id: transcriptOutput, //variable id
-//     emailAddress: submittedAddress,
-//     password: submittedPassword,
-//     diagramID: submittedDiagramID,
-//     //???????
-//     //prompt: activeTab,
-//   });
-
-//   //create the fetch request
-//   fetch(`http://localhost:8080/api/v1/transcripts/storeAccount`, {method: "POST", headers: headerData, body: fullEntry})
-//   .then((response) => response.json())
-//   .then((data) => {
-//     console.log(data);
-//   });
-
-//   //empty the textboxes (maybe only do this if the submission fails do to an error or no transcripts or something else?)
-//   document.getElementById("emailAddress").value = "";
-//   document.getElementById("password").value = "";
-//   document.getElementById("diagramID").value = "";
-// }
-
-export async function getBestIntents(){
-
-  // apparently this has to be initialized to a certain type, or else an error is caused.
-  // this way, i can change the value in the .then below i think
-  // scratch that, its just the .then scope is screwing up.
-  var intents = ["default intent 1", "default intent 2", "default intent 3"];
 
   //create the fetch request
-  //use threeIntents for now. Hopefully in the future it will be a new getBestIntents controller.
-  intents = await
-  fetch(`http://localhost:8080/api/v1/transcripts/threeIntents`, {method: "GET", headers: headerData})
+  fetch(`http://localhost:8080/api/v1/transcripts/createBlock`, {method: "POST", headers: headerData, body: fullEntry})
```

Comment:
We need to change the base of the routes for the backend calls for deployment, here is just one spot.